packages/compass-sidebar/src/components/favorite-button/favorite-button.jsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import FontAwesome from 'react-fontawesome';
+import {
+  mergeProps,
+  useFocusRing
+} from '@mongodb-js/compass-components';
+
+import styles from './favorite-button.module.less';
+
+const FavoriteButton = ({
+  favoriteOptions,
+  toggleIsFavoriteModalVisible
+}) => {
+  const isFavorite = !!favoriteOptions;
+  const fontAwesomeName = isFavorite ? 'star' : 'star-o';
```

Comment:
Don't we have this in `compass-components` now?

Reply:
Ah good thinking did just add that. Updated